acceptance/config/nodes/huaweios-6-powerpc.yaml

```diff
@@ -0,0 +1,14 @@
+---
+HOSTS:
+  huaweios-6-powerpc:
+    roles:
+      - agent
+    platform: huaweios-6-powerpc
+    hypervisor: none
+    ip: 10.0.7.26
+    vmhostname: huawei-ce6850-2-debian-vm-eth0.ops.puppetlabs.net
+CONFIG:
```

Comment:
Agreed, looks like CONFIG doesn't need to be there. Look at the other configs for comparable examples.